packages/lodestar-cli/test/unit/lodestar/account.test.ts

```diff
@@ -0,0 +1,114 @@
+import yargs from "yargs/yargs";
+import { expect } from "chai";
+import { stub, match } from "sinon";
+
+import { rootDir } from "../../constants";
+import { account } from "../../../src/cmds/account";
+import { ValidatorDirManager } from "../../../src/validatorDir";
+import { getAccountPaths } from "../../../src/cmds/account/paths";
+import { init } from "../../../src/cmds/init";
+import { TestnetName } from "../../../src/testnets";
+import { medallaConfig } from "../../../src/testnets/medalla";
+import { existsSync, readFileSync, readdirSync, unlinkSync } from "fs";
+
+describe.only("account cli", function() {
+  this.timeout("10 min");
```

Comment:
A unit test should be very quick. If you do something that takes more than seconds move it to e2e tests

Reply:
The reason for this timeout is connected to the reason for the timeout for init below.  By default, an individual unit test function will run for 2000ms, and the beacon init takes 6000m (as estimated in the Promise resolve timeout below)